jest.config.js

```diff
@@ -129,7 +129,7 @@ module.exports = {
   // snapshotSerializers: [],
 
   // The test environment that will be used for testing
-  testEnvironment: "node",
+  testEnvironment: "jest-environment-node",
```

Comment:
I'm curious: is this change strictly necessary?

I tried reverting this change locally and tests did pass.

Reply:
Yeah this is now working for me too. Great catch! Thanks!